integrationtests/selector/integrationtest_selector.py

```diff
@@ -922,14 +922,15 @@ def test_get_available_labels(reset_after_trigger: bool):
         )
     )
 
-    selector.inform_data(
+    selector.inform_data_and_trigger(
```

Comment:
I suppose this was an error and the change in the abstract selection strategy fixed this :)?

Reply:
Exactly :)

Reply:
And updated again :)